### PR TITLE
Feature/rtenu-53 accessibility tools

### PR DESCRIPTION
**Pull request by saneDG**
- Add [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y)
- Configure accessibility settings. Set as "recommended".

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "extends": ["prettier"],
-  "plugins": ["prettier"],
+  "extends": ["prettier", "plugin:jsx-a11y/strict"],
```
Comment (NinaDang97) [inline .eslintrc:2]: `"plugin:jsx-a11y/recommended"` would be better and less restrictive for a starting point

Comment (mehtis) [reply .eslintrc:2]: How strict is strict? Because we're doing software that needs to be accessible, strict rules might be what we should be using.

Comment (NinaDang97) [reply .eslintrc:2]: `recommended` stills notifies us with similar accessibility errors that `strict` does, except `recommended` gives us options to customize in a few cases (ref: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y#rule-strictness-in-different-modes)

Comment (saneDG) [reply .eslintrc:2]: Idea here was to try strict rules first and we can fallback to recommended if they unnecessarily slow development down. It's easier to go that way than tighten the rules after changes has been applied.

Comment (mehtis) [reply .eslintrc:2]: Checked a few other repos, and dvk seems to be using recommended. So based on that I'd say that it should be enough 🙂

Comment (saneDG) [inline .eslintrc:2]:
```suggestion
  "extends": ["prettier", "plugin:jsx-a11y/recommended"],
```

Comment (saneDG) [reply .eslintrc]: Changed setting to recommended. Thanks for mentioning this @NinaDang97!

**Review by mehtis — APPROVED** [review]
You should add the dash to the PR name (e.g. rentu-53 here) so that the PR can be more easily found later on/linking to Jira will work. It's annoying that it has to be done each time, but it'll be of help in the long run.

**Pull request merged**